backend/src/users/users.service.ts

```diff
@@ -0,0 +1,88 @@
+import { ConflictException, Injectable, NotFoundException } from '@nestjs/common';
+import {InjectModel} from '@nestjs/mongoose'
+import { Model } from 'mongoose';
+import { CreateUserDto } from './dto/create-user.dto';
+import { UpdateUserDto } from './dto/update-user.dto';
+import {User, UserDocument} from './schemas/user.schema';
+
+@Injectable()
+export class UsersService {
+  constructor(@InjectModel(User.name) private userModel: Model<UserDocument>) {}
+
+  async create(createUserDto: CreateUserDto): Promise<UserDocument> {
```

Comment:
We may want to consider removing the create method from the users resource and handle creation/registration specifically through the auth resource or vice versa. Unless we do want to keep both? since they return different data